chapters/packages.tex

```diff
@@ -63,22 +63,22 @@ \subsection{Importing Definitions from a Package}\doublelabel{importing-definiti
 %TODO-FORMAT Should be formatted using tabs or tabular?
 An \lstinline!import!-clause can occur in one of the following syntactic forms:
 
-% Note a syntactic form cannot be written as a single \lstinline[mathescape=true] due to LaTeXML problem reported here:
+% Note a syntactic form cannot be written as a single \lstinline due to LaTeXML problem reported here:
 %   https://github.com/brucemiller/LaTeXML/issues/1274 (marked as fixed as of commit 80d7940)
 % Instead, we need to break up the \lstinline at each piece of math (taking the opportunity to leave the whitespace
 % outside both \lstinline and math).
 
-\lstinline[mathescape=true]!import $\mathit{definitionname}$;! (qualified import of top-level definition)
+\lstinline!import $\mathit{definitionname}$;! (qualified import of top-level definition)
 
-\lstinline[mathescape=true]!import $\mathit{packagename}$.$\mathit{definitionname}$;! (qualified import)
+\lstinline!import $\mathit{packagename}$.$\mathit{definitionname}$;! (qualified import)
```

Comment:
Currently I can only get it to work if I write:
`\lstinline!import $\mathit{packagename}$.!\lstinline!$\mathit{definitionname}$;! (qualified import)`

Reply:
Right, seems like we've been hit by https://github.com/brucemiller/LaTeXML/issues/1274 again.  The issue exists also on _master_, but I'll fix it here regardless since you mentioned it.

Reply:
Fixed.  Resolving.